src/redux/data.ts

```diff
@@ -57,17 +57,30 @@ export interface IncomeW2 {
 }
 
 export enum Income1099Type {
+  B = 'B',
```

Comment:
I think the strings here should be changed to 1099-B and 1099-INT for clarity

Reply:
Fixed this in the drop down code text mappign